shell/platform/darwin/ios/BUILD.gn

```diff
@@ -74,6 +74,9 @@ source_set("flutter_framework_source") {
   deps = []
 
   sources = [
+    # iOS embedder is migrating to ARC.
```

Comment:
I'm still a little nervous about sending the announcement since we haven't done any prod code migration to be sure we can truly start migrate. There could be a (small) chance that we'd have to fix some more stuff before the migration can really start. 

I added this comment here so people would probably notice when they are trying to add new mrc files.